docs/_static/css/custom.css

```diff
@@ -52,3 +52,13 @@ div.sponsor-div-box {
    font-size: 0.9rem;
    text-align: center;
  }
+
+ /* Fix version switcher button margin bottom */
+ button.version-switcher__button{
+   margin-bottom: 0em;
+ }
+
+ /* Fix scrollbars on primary sidebar */
+ .bd-sidebar-primary .sidebar-end-items{
+   margin-top: unset;
+ }
```

Comment:
This custom CSS block will remove the scrollbars. I was able to test on Chrome, Edge and Firefox. I think it will work on Safari as well, but please check and let me know @hildeweerts.

Reply:
Unfortunately the horizontal scrollbar is still there in Safari, though the vertical one is now only there when it's actually necessary, so that's nice!

![image](https://user-images.githubusercontent.com/24417440/203566906-b13d716b-1a22-4ce3-b975-99e9133184e7.png)

Reply:
I did try to look into this using BrowserStack. It seems the issue is only on the latest Safari build for macOS Ventura; I am not sure which version you are using. 

Unfortunately, the access for the free account lasts for only a minute so I was not able to check much. If you can inspect the padding using the Inspect Tool, that might help us figure out where we need to make a change.

Reply:
Thanks for checking - I'm indeed using Safari on Ventura. It's not *super* disturbing and sounds more like a Safari + pydata theme issue that might be resolved over time.